src/v/cluster/topic_updates_dispatcher.cc

```diff
@@ -125,17 +125,6 @@ ss::future<std::error_code> topic_updates_dispatcher::apply(
     if (ec == errc::success) {
         add_allocations_for_new_partitions(
           assignments, get_allocation_domain(tp_ns));
-        ss::chunked_fifo<ntp_leader> leaders;
```

Comment:
Also has to be removed from `topic_updates_dispatcher::apply_snapshot` I guess

Reply:
right, i need to do it